Linux.md

```diff
@@ -96,3 +101,21 @@
 
 ### <a name="moveCursorToTheEndOfLine"></a> Moves the cursor to the end of the line
 `Ctrl+e`
+
+### <a name="lastSudo"></a> Runs Previous Sudo command
+`$!!`
+
+### <a name="burnIt"></a> ASCII art animation that generates a burning fire on the terminal
+`aafire`
+
+### <a name="showSW"></a> Watch Star Wars in ASCII on terminal
+`telnet towel.blinkenlights.nl`
+
+### <a name="dnsip"></a> DNS lookup / Find IP of a website
+`dig websitename`
+
+### <a name="randomFake"></a> Generate random and possible fake identites
+`rig`
+
+### <a name="showCpu"></a> Show CPU information and architecture
```

Comment:
Even this

Reply:
I made the changes as per your sugggestion review them @arverma007